### PR TITLE
Conversions: Added length units and fixed pressure units

### LINKED ISSUE #3872: Conversions: Ability to convert µm to mil
Example:

**0.1 µm in mil**
[https://duckduckgo.com/?q=0.1%20%C2%B5m%20in%20mil&kp=1&kad=en_US&kl=wt-wt](https://duckduckgo.com/?q=0.1%20%C2%B5m%20in%20mil&kp=1&kad=en_US&kl=wt-wt)

------
IA Page:  http://duck.co/ia/view/conversions 
[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @mintsoft

### PR DESCRIPTION
**Pull request by nalinbhardwaj**
## Description of new Instant Answer, or changes

- Added prefix + `pascal` and `pascals` in all pressure aliases
- Removed pa and pas from pressure `bar`
- Added micrometre
- Added nanometre(later commits)
- Added picometre

## Related Issues and Discussions

Fixes #3872, #3873 and #3876.

## People to notify

@bsstoner @mintsoft 

---

Instant Answer Page: https://duck.co/ia/view/conversions

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by nalinbhardwaj** [comment]
Sorry about that. Cleaned up the new units now to match the other unit style.

File: share/goodie/conversions/ratios.yml [hunk 1]
```diff
@@ -892,6 +892,8 @@ plural: leap years
 aliases:
   - pa
   - pas
+  - pascal
```
Comment (mintsoft) [inline share/goodie/conversions/ratios.yml]: Can you remove pascal, as it's the name of the unit; it shouldn't be specified in both

File: share/goodie/conversions/ratios.yml [hunk 2]
```diff
@@ -900,6 +902,8 @@ plural: pascals
 aliases:
   - kpa
   - kpas
+  - kilopascal
```
Comment (mintsoft) [inline share/goodie/conversions/ratios.yml]: Same here

File: share/goodie/conversions/ratios.yml [hunk 3]
```diff
@@ -916,14 +922,16 @@ plural: megapascals
 aliases:
   - gpa
   - gpas
+  - gigapascal
```
Comment (mintsoft) [inline share/goodie/conversions/ratios.yml]: These two should be unnecessary as the unit name and plural are used in addition to the aliases

File: share/goodie/conversions/ratios.yml [hunk 4]
```diff
@@ -916,14 +922,16 @@ plural: megapascals
...
+  - gigapascals
 factor: 1e-09
 type: pressure
 unit: gigapascal
 plural: gigapascals
 ---
 aliases:
-  - pa
-  - pas
+  - bar
```
Comment (mintsoft) [inline share/goodie/conversions/ratios.yml]: Same again; although I don't know why pa was in there, as that's Pascal, not bar so it was already wrong

File: share/goodie/conversions/ratios.yml [hunk 5]
```diff
@@ -1638,3 +1646,23 @@ factor: 1.943844492440605
 type: speed
 unit: knot
 plural: knots
+---
+aliases:
```
Comment (mintsoft) [inline share/goodie/conversions/ratios.yml]: Can you move these into place where the other units of length are defined for organisation

**Comment by mintsoft** [comment]
@nalinbhardwaj Also can you add a couple of tests for the new units, doesn't need full coverage just that they exist

**Comment by nalinbhardwaj** [comment]
Hey @mintsoft. I'll be making a commit with those changes in a short while. While I'm at it, I was thinking to add nanometre as well. But `nm` seems to being used for nautical miles. Is there any way to specify for only `NM` to treated as nautical miles?

**Comment by mintsoft** [comment]
@nalinbhardwaj At the moment, the units are unfortunately (and inaccurately!) case insensitive; I'm working on tidying this up at the moment, but it's reasonably involved so it's probably better leaving that for now.

**Comment by nalinbhardwaj** [comment]
@mintsoft Cool. Left the changes for now. Check new changes. Added tests for all new units.

**Comment by mintsoft** [comment]
@nalinbhardwaj No problem at all, this is why we have the review! Welcome 👍

**Comment by mintsoft** [comment]
Thanks for the merge @bsstoner

**Pull request merged**